### ISSUE TITLE
How can I use with multiple link socket server?

### ISSUE THREAD

**Issue by henrytom1703** [probil/vue-socket.io-extended#245]

**Comment by probil** [comment]
Hi @henrytom1703
Thanks for your issue
It's not officially supported yet but you can try to use plugin multiple times with different prefixes and different servers.
I mean something like this:
```js
import VueSocketio from 'vue-socket.io-extended';
import io from 'socket.io-client';

const socket1 = io('http://socketserver1.com');
const socket2 = io('http://socketserver2.com');

Vue.use(VueSocketio, socket1, {
  actionPrefix: 'socket1_',
  mutationPrefix: 'SOCKET1_',
});
Vue.use(VueSocketio, socket2, {
  actionPrefix: 'socket2_',
  mutationPrefix: 'SOCKET2_',
});
```

And after that you can use them in your Vuex stores this way
```js
export default new Vuex.Store({
  state: {
    messageFromSocket1: null,
    messageFromSocket2: null,
  },
  mutations: {
    SOCKET1_USER_MESSAGE(state, message) {
      state.messageFromSocket1 = message;
    },
    SOCKET2_USER_MESSAGE(state, message) {
      state.messageFromSocket2 = message;
    },
  },
  actions: {
    socket1_userMessage({ commit, dispatch }, message) {
      // ... handle action from socket1
    },
    socket2_userMessage({ commit, dispatch }, message) {
      // ...handle action from socket2
    },
  },
})
```
In theory it should work. Let me know when you try it. I will be glad to add it to the README

**Comment by probil** [comment]
Closed because it duplication of #81
